tests/full-version-label.sh

```diff
@@ -3,60 +3,18 @@ set -eo pipefail
 # The purpose of this test is to ensure that the output of the "nodeos --full-version" command matches the version string defined by our CMake files
 echo '##### Nodeos Full Version Label Test #####'
 # orient ourselves
-[[ "$EOSIO_ROOT" == '' ]] && EOSIO_ROOT=$(echo $(pwd)/ | grep -ioe '.*/eos/')
-[[ "$EOSIO_ROOT" == '' ]] && EOSIO_ROOT=$(echo $(pwd)/ | grep -ioe '.*/EOSIO/eosio/')
-[[ "$EOSIO_ROOT" == '' ]] && EOSIO_ROOT=$(echo $(pwd)/ | grep -ioe '.*/build/' | sed 's,/build/,,')
+[[ "$EOSIO_ROOT" == '' ]] && EOSIO_ROOT=$(pwd)
```

Comment:
We should call this `BUILD_ROOT`, as `EOSIO_ROOT` is used [elsewhere](https://github.com/EOSIO/eos/blob/9f4f242f27eacb4c1557ff82ae95d320c0031b32/CMakeModules/eosio-config.cmake.in#L1) in this repo to refer to the root of the EOSIO repository.